{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -198,6 +197,33 @@
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 
+
+# AWS S3 config
+AWS_ACCESS_KEY_ID = env.str("AWS_ACCESS_KEY_ID", "")
+AWS_SECRET_ACCESS_KEY = env.str("AWS_SECRET_ACCESS_KEY", "")
+AWS_STORAGE_BUCKET_NAME = env.str("AWS_STORAGE_BUCKET_NAME", "")
+AWS_STORAGE_REGION = env.str("AWS_STORAGE_REGION", "")
+
+USE_S3 = (
+    AWS_ACCESS_KEY_ID and
+    AWS_SECRET_ACCESS_KEY and
+    AWS_STORAGE_BUCKET_NAME and
+    AWS_STORAGE_REGION
+)
+
+if USE_S3:
+    AWS_S3_CUSTOM_DOMAIN = env.str("AWS_S3_CUSTOM_DOMAIN", "")
+    AWS_S3_OBJECT_PARAMETERS = {"CacheControl": "max-age=86400"}
+    AWS_DEFAULT_ACL = env.str("AWS_DEFAULT_ACL", None)
```

Comment:
`AWS_DEFAULT_ACL` needs a default value. See this discussion: https://discuss.crowdbotics.com/t/getting-access-denied-on-s3-storage/756/2

Reply:
Done 6179387